packages/apps/dashboard/ui-2024/src/features/Leaderboard/components/ChainCell.tsx

```diff
@@ -2,10 +2,16 @@ import { Typography } from '@mui/material';
 import { NetworkIcon } from '@components/NetworkIcon';
 import { getNetwork } from '@utils/config/networks';
 
-export const ChainCell = ({ chainId }: { chainId: number }) => (
+export const ChainCell = ({
+  chainId,
+  isMobile,
+}: {
+  chainId: number;
+  isMobile: boolean;
+}) => (
   <Typography
     variant="body1"
-    whiteSpace="nowrap"
+    whiteSpace={isMobile ? 'wrap' : 'nowrap'}
```

Comment:
It was noticed on mobile, but in case of really long network name we will have it on desktop as well, so maybe we apply `wrap` for both, wdyt @adrian-oleskiewicz ?

Reply:
Good point @dnechay, done